ServiceWebsite/ServiceWebsite.AcceptanceTests/TestClients/BookingsApiTestClient.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using RestSharp;
+using ServiceWebsite.BookingsAPI.Client;
+using ServiceWebsite.Common;
+
+namespace ServiceWebsite.AcceptanceTests.TestClients
+{
+    public class BookingsApiTestClient : ApiTestClientBase
+
+    {
+        public BookingsApiTestClient(RestClient client, string token) : base(client, token)
+        {
+        }
+
+        public HttpStatusCode CreateNewVideoHearingsBooking(BookNewHearingRequest requestBody)
+        {
+            var request = Post("/hearings", requestBody);
```

Comment:
Like I said above, you can change these methods to return the entire response from the base class or remove the base class and do everything here.